presentation/src/main/java/com/keeply/presentation/ui/onboarding/OnboardingScreen.kt

```diff
@@ -0,0 +1,161 @@
+import com.kakao.sdk.user.UserApiClient
+import com.keeply.presentation.R
+import com.keeply.presentation.core.components.KeeplyButton
+import com.keeply.presentation.core.theme.KeeplyTheme
+import com.keeply.presentation.ui.onboarding.component.KakaoLoginButton
+import com.keeply.presentation.ui.onboarding.component.OnboardingPager
+import org.orbitmvi.orbit.compose.collectAsState
+
+@Composable
+fun OnboardingRoute(
+    viewModel: OnboardingViewModel = hiltViewModel()
+) {
+    val uiState by viewModel.collectAsState()
+
+    OnboardingScreen(
+        onboardingPage = uiState.onboardingPage,
+        updateOnboardingPage = viewModel::updateOnboardingPage
+    )
+}
+
+@Composable
+fun OnboardingScreen(
+    onboardingPage: OnboardingPage,
+    updateOnboardingPage: (OnboardingPage) -> Unit = {}
+) {
+    val context = LocalContext.current
+
+    Box(
+        modifier = Modifier
+            .fillMaxSize()
+            .background(KeeplyTheme.colors.neutral100)
+    ) {
+        OnboardingPager(
+            modifier = Modifier
+                .padding(top = 40.dp)
+                .width(246.dp)
+                .align(Alignment.TopCenter),
+            onboardingPage = onboardingPage
+        )
+
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+                .height(270.dp)
+                .align(Alignment.BottomCenter)
+                .background(
+                    brush = Brush.verticalGradient(
+                        colors = listOf(
+                            Color.Transparent,
+                            KeeplyTheme.colors.neutral100,
+                            KeeplyTheme.colors.neutral100
+                        )
+                    )
+                )
+        )
+
+        val bottomModifier = Modifier
+            .padding(
+                start = 16.dp,
+                end = 16.dp,
+                bottom = 50.dp
+            )
+            .fillMaxWidth()
+            .align(Alignment.BottomCenter)
+
+        if (onboardingPage != OnboardingPage.THIRD) {
+            KeeplyButton(
+                modifier = bottomModifier,
+                text = stringResource(R.string.next),
+                onClick = { updateOnboardingPage(onboardingPage) }
+            )
+        } else {
+            KakaoLoginButton(
+                modifier = bottomModifier,
+                onClick = {
+                    loginWithKakaoTalk(context)
+                }
+            )
+        }
+    }
+}
+
+private fun loginWithKakaoTalk(
+    context: Context,
+) {
+    val callback: (OAuthToken?, Throwable?) -> Unit = { token, error ->
+        if (error != null) {
+            error.printStackTrace()
+            Log.e("user", "실패")
+        } else if (token != null) {
+            UserApiClient.instance.me { user, _ ->
+                Log.e("token", token.toString())
+                Log.e("user", user.toString())
+            }
+        }
+    }
+    // 카카오톡이 설치되어 있으면 카카오톡으로 로그인, 아니면 카카오계정으로 로그인
```

Comment:
p3: 앱/웹 표현을 써도 좋을 것 같아요
```suggestion
    // 카카오톡이 설치되어 있으면 앱으로 로그인, 아니면 웹으로 로그인
```

Reply:
수정 반영했어요~